Change Helm reference `--set` formatting

ptgott opened the pull request:
Closes #9686

Since the `--set` flag examples aren't commands, this change removes the shell prompts and changes code fence labels from `code` to `text`.

ptgott committed:
Change Helm reference `--set` formatting

Closes #9686

Since the `--set` flag examples aren't commands, this change removes the
shell prompts and changes code fence labels from `code` to `text`.

hugoShaka commented:
Could we just drop them? This syntax is awkward, does not suit well complex configurations and gives us more maintenance work.

ptgott committed:
Remove --set examples from Helm references

Keep the values.yaml examples. Used the following script to generate the
change:

files="$(find docs/pages/reference/helm-reference -name "*.mdx")";

for i in $(echo "$files"); do
    tmp=$(awk 'BEGIN{yaml=""}
    /<Tabs>/{t=1}
    t==1 && /```yaml/{y=1; yaml="`values.yaml` example:\n\n"}
    y==1{yaml = (yaml "\n" $0)}
    y==1 && /```$/{y=0; print yaml; yaml=""}
    t==1 && /<\/Tabs/{t=0; next}
    t==0{print $0 }' $i)
    echo "$tmp" > $i;
done

ptgott commented:
@hugoShaka Sounds good! I've made the change in [3a7472a](https://github.com/gravitational/teleport/pull/24944/commits/3a7472a72c558c7d3a93cdca1c54b8a6c09d0ab3)

hugoShaka reviewed (approved):
🎉
